CONTRIBUTING.md

```diff
@@ -28,6 +28,23 @@ cp config-defaults.yaml cos-alerter.yaml
 docker run -p 8080:8080 --rm --mount type=bind,source="$(pwd)"/cos-alerter.yaml,target=/etc/cos-alerter.yaml,readonly -it cos-alerter:0.2.0
 ```
 
+## Run With Kubernetes
+Prepare the image:
+```shell
+rockcraft pack
+# update <registry-ip> with the actual IP of your docker registry
+# update <image-tag> with the image tag you would like to use in testing
+skopeo copy oci-archive:cos-alerter_0.8.0_amd64.rock docker://<registry-ip>/<image-tag> --dest-tls-verify=false
+```
+
+Run:
+```shell
+# in k8s-local-test/deploy.yaml update <registry-ip>/<image-tag> with appropriate values
+# in k8s-local-test/deploy.yaml update cos-alerter.yaml configmap with appropriate values
+kubectl apply -f k8s-local-test/deploy.yaml
+kubectl apply -f k8s-local-test/svc.yaml
+```
+
```

Comment:
Reason I committed this was just to help out in local testing hence the dir name, having opinionated k8s-manifests as a deployment option wouldn't be an added value imo cause it really depends on the user preference of how he wants to deploy it on k8s.

I've left steps intentionally in the contributing section only to highlight it's just for testing (no need to expose the IP as cos-alerter will be on the same k8s cluster as COS/alertmanager), if this doesn't really add any value and will only cause confusion I think it will be wise to remove it, wdyt??

Reply:
My point is that if it is only for local testing, maybe we don't need to commit it to the repo? I kind of think this doesn't belong here.